changelog/fragments/1749470526-lock-update-marker.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
```

Comment:
Do we need a changelog entry for this change?  It doesn't seem like a user-facing change, more an internal detail?

Reply:
As it introduces a locking mechanism for the update marker I prefer to have it on the release notes, it could give a clue in case some deadlock or some other issue crop up.  
If there was a flag to disable this and I could activate it only when the feature is complete I probably wouldn't add a changelog entry.